docs/bibliography/diffusion-mri.md

```diff
@@ -1,18 +1,32 @@
 # Resources for learning the basics of diffusion MRI
 
-## Here as some basic readings to introduuce the field
+## Here are some readings to introduuce the field
 
-Wandell BA (2016) Clarifying Human White Matter. Annu Rev Neurosci 39(1):annurev-neuro-070815-013815.
+**A great overview of the field:**
 
-## Automated Fiber Quantification is a method for analyzing white matter tracts
+    Wandell BA (2016) Clarifying Human White Matter. Annu Rev Neurosci 39(1):annurev-neuro-070815-013815.
 
-Here is the paper that introducued the method:
+**Review paper on fiber tractography methods:**
+
+    Jeurissen B, Descoteaux M, Mori S, Leemans A (2019) Diffusion MRI fiber tractography of the brain. NMR Biomed 32(4):1–22.
+
+**Two papers from our group focused on using diffusion MRI to study brain development and plasticity:**
+
+    Huber E, Donnelly PMPM, Rokem A, Yeatman JD (2018) Rapid and widespread white matter plasticity during an intensive reading intervention. Nat Commun 9(1):2260.
```

Comment:
Does Patrick really have four initials?

Reply:
No but I can't figure out how to resolve this through the GUI

Reply:
It should be Donnelly PM

Reply:
OK - I think that I just resolved that one.